packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverCarousel/TeachingPopoverCarousel.types.ts

```diff
@@ -1,112 +1,51 @@
-import * as React from 'react';
-import { ComponentProps, ComponentState, EventData, EventHandler, Slot } from '@fluentui/react-utilities';
-import { Button } from '@fluentui/react-button';
+import { ComponentProps, ComponentState, EventHandler, Slot } from '@fluentui/react-utilities';
 import { PopoverContextValue } from '@fluentui/react-popover';
-import { TeachingPopoverCarouselNavProps } from '../TeachingPopoverCarouselNav/index';
+
+import { type CarouselContextValue } from './Carousel/CarouselContext';
+import type { CarouselPageChangeData } from './Carousel/Carousel.types';
 
 export type TeachingPopoverCarouselSlots = {
   /**
    * The element wrapping carousel pages and navigation.
    */
   root: NonNullable<Slot<'div'>>;
-
-  /**
-   * The element wrapping the navigation of the carousel.
-   */
-  footer: NonNullable<Slot<'div'>>;
-
-  /**
-   * The previous button slot.
-   */
-  previous: Slot<typeof Button>;
-
-  /**
-   * The next button slot.
-   */
-  next: NonNullable<Slot<typeof Button>>;
-
-  /**
-   * The page nav slot when using icon navigation.
-   */
-  nav: Slot<TeachingPopoverCarouselNavProps>;
-
-  /**
-   * The page count text slot for paginationType text
-   */
-  pageCount: Slot<'div'>;
-};
-
-export type TeachingPopoverCarouselLayout = 'offset' | 'centered';
-
-// For localization or customization, users may want to modify this for their own purposes
-export type TeachingPopoverPageCountChildRenderFunction = (currentPage: number, totalPages: number) => React.ReactNode;
-export type TeachingPopoverPageCountRenderType = TeachingPopoverPageCountChildRenderFunction | string;
-export type TeachingPopoverStrings = {
-  previous: string;
-  initialStepText: string;
-  next: string;
-  finalStepText: string;
-  /**
-   * Localize the separator between numbers, or use a function to fully override
-   */
-  pageCountText?: TeachingPopoverPageCountRenderType;
-};
-
-export type TeachingPopoverPageChangeData = EventData<'click', React.MouseEvent<HTMLButtonElement>> & {
-  /**
-   * The page to be set after event has occurred.
-   */
-  currentPage: number;
 };
 
 /**
  * TeachingPopoverCarousel Props
  */
-export type TeachingPopoverCarouselProps = Partial<ComponentProps<TeachingPopoverCarouselSlots>> & {
+export type TeachingPopoverCarouselProps = ComponentProps<TeachingPopoverCarouselSlots> & {
   /**
-   * Controls whether buttons will be centered (balanced) or right aligned
-   * Defaults to 'centered'.
+   * The initial page to display in uncontrolled mode.
    */
-  layout?: TeachingPopoverCarouselLayout;
+  defaultValue?: string;
 
   /**
-   * Dictates whether pagination uses text or icons
-   * Defaults to icons
+   * The value of the currently active page.
    */
-  paginationType?: 'text' | 'icon';
+  value?: string;
```

Comment:
The callback is called `onPageChange`  we use `value` to control the carousel. Let's make this more consistent and either use `onValueChange` or use `page` and `defaultPage` instead